src/Microsoft.ML.Data/Transforms/NormalizeUtils.cs

```diff
@@ -11,6 +11,7 @@
 using Microsoft.ML.Runtime.Model.Pfa;
 using Newtonsoft.Json.Linq;
 using System.Collections.Generic;
+using Microsoft.ML.Data.StaticPipe.Runtime;
```

Comment:
>using Microsoft.ML.Data.StaticPipe.Runtime [](start = 0, length = 42)

why?

Reply:
I have two namespaces, one for types I expect users to declare directly (without `Runtime`), and the other I don't expect them to use. I think this is appropriate, but I put the `PipelineColumn` instances in there, which I think was probably a mistake. I'd rather handle whatever changing of namespaces are necessary though in a PR tha tis not this PR, since this is pretty insularly about normalizers which I think is appropriate.

Reply:
Here's a practical problem showing why I made a mistake to put the `PipelineColumn` in a separate namespace... here's with them included in intellisense.

![image](https://user-images.githubusercontent.com/8295757/45187545-de4cc380-b1e5-11e8-8e3a-cd25732a2000.png)

The same without them included, now contains fully qualified name.

![image](https://user-images.githubusercontent.com/8295757/45187574-ffadaf80-b1e5-11e8-8735-29c9fb6a0068.png)